### ISSUE TITLE
[Bug]: Latest update of mm_sd_v15_v2.ckpt throws error, generates noise, deconfigures automatic 1111

### ISSUE THREAD

**Issue by jlpintado** [continue-revolution/sd-webui-animatediff#217]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### Have you read FAQ on README?

- [X] I have updated WebUI and this extension to the latest version

### What happened?

In the recent update, when using the mm_sd_v15_v2.ckpt model, if I configure generating video and png image sequence, in the first time it will generate only the png image sequence, a second time, it generates image sequence that is only noise. Then I disable AnimateDiff and generate an image, it generates noise, in order to generate an image I must restart automaticc 1111. This problem is only generated when using mm_sd_v15_v2.ckpt, I have updated it several times. If I first use other models like temporaldiff-v1-animatediff.ckpt or the same mm_sd_v15.ckpt they work fine.
![00000-684530869](https://github.com/continue-revolution/sd-webui-animatediff/assets/77060047/1e8ee724-d40b-4677-96ef-e115286e994c)

### Steps to reproduce the problem

1- First attempt to generate video with AnimateDiff with the model mm_sd_v15_v2.ckpt mode, only a sequence of images is generated.
2 - Second attempt only generates noise

### What should have happened?

mm_sd_v15_v2.ckpt (If I use other models I don't have the problem)

### Commit where the problem happens

version: [1.6.0]•  python: 3.10.6  •  torch: 2.0.1+cu118  •  xformers: 0.0.20  •  gradio: 3.41.2  •  checkpoint: [06513b0322]
- Control net : ControlNet v1.1.410
- Cross attention optimization = sdp - scaled dot product / xformers (With both options I get the same error)

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
No
```

### Console logs

```Shell
To create a public link, set `share=True` in `launch()`.
Startup time: 23.8s (prepare environment: 13.9s, import torch: 3.5s, import gradio: 1.0s, setup paths: 1.0s, initialize shared: 0.3s, other imports: 0.7s, setup codeformer: 0.1s, load scripts: 2.3s, create ui: 0.6s, gradio launch: 0.3s).
Creating model from config: E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\configs\v1-inference.yaml
Applying attention optimization: sdp... done.
Model loaded in 5.1s (load weights from disk: 2.1s, create model: 0.3s, apply weights to model: 0.7s, load textual inversion embeddings: 0.9s, calculate empty prompt: 0.9s).
2023-10-13 23:03:11,792 - AnimateDiff - INFO - AnimateDiff process start.
2023-10-13 23:03:11,793 - AnimateDiff - INFO - You are using tested mm mm_sd_v15_v2.ckpt.
2023-10-13 23:03:11,793 - AnimateDiff - INFO - Loading motion module mm_sd_v15_v2.ckpt from E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\extensions\sd-webui-animatediff\model\mm_sd_v15_v2.ckpt
2023-10-13 23:03:14,509 - AnimateDiff - WARNING - Missing keys <All keys matched successfully>
2023-10-13 23:03:15,446 - AnimateDiff - INFO - Injecting motion module mm_sd_v15_v2.ckpt into SD1.5 UNet middle block.
2023-10-13 23:03:15,446 - AnimateDiff - INFO - Injecting motion module mm_sd_v15_v2.ckpt into SD1.5 UNet input blocks.
2023-10-13 23:03:15,447 - AnimateDiff - INFO - Injecting motion module mm_sd_v15_v2.ckpt into SD1.5 UNet output blocks.
2023-10-13 23:03:15,447 - AnimateDiff - INFO - Setting DDIM alpha.
2023-10-13 23:03:15,448 - AnimateDiff - INFO - Injection finished.
2023-10-13 23:03:15,448 - AnimateDiff - INFO - Hacking lora to support motion lora
*** Error running before_process: E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\extensions\sd-webui-animatediff\scripts\animatediff.py
    Traceback (most recent call last):
      File "E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\modules\scripts.py", line 611, in before_process
        script.before_process(p, *script_args)
      File "E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\extensions\sd-webui-animatediff\scripts\animatediff.py", line 52, in before_process
        self.lora_hacker.hack()
      File "E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\extensions\sd-webui-animatediff\scripts\animatediff_lora.py", line 23, in hack
        import network
    ModuleNotFoundError: No module named 'network'

---
100%|██████████████████████████████████████████████████████████████████████████████████| 25/25 [01:08<00:00,  2.72s/it]
*** Error running postprocess: E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\extensions\sd-webui-animatediff\scripts\animatediff.py
    Traceback (most recent call last):
      File "E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\modules\scripts.py", line 651, in postprocess
        script.postprocess(p, processed, *script_args)
      File "E:\1 j1 STABLE ACTUALIZADO\stable-diffusion-webui-master\extensions\sd-webui-animatediff\scripts\animatediff.py", line 72, in postprocess
        self.cn_hacker.restore()
    AttributeError: 'NoneType' object has no attribute 'restore'

---
Total progress: 100%|██████████████████████████████████████████████████████████████████| 25/25 [01:10<00:00,  2.82s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 25/25 [01:10<00:00,  2.70s/it]
```

### Additional information

Before the animediff update it works perfectly.

**Comment by zappityzap** [comment]
The network module is part of Automatic1111. You might check if the built-in Lora extension is enabled.

**Comment by jlpintado** [comment]
Hello @zappityzap , I don't know if this answers your question, I apologize because I don't master this, but I use loras normally, they work well for me, that is, it seems to me that they are enabled. Also download the camera movement Lora from animediff and place them on the lora models.
![3563](https://github.com/continue-revolution/sd-webui-animatediff/assets/77060047/9c7a1da2-d9de-4b29-887a-a976724e3216)

**Comment by zappityzap** [comment]
On the Automatic1111 Extensions tab, scroll to the bottom and look for the Lora extension:
![image](https://github.com/continue-revolution/sd-webui-animatediff/assets/128872140/9a7fdb33-e346-4bfe-81fd-16e127a2bf0e)

**Comment by jlpintado** [comment]
@zappityzap Thank you very much for the clarification, here it is.
![t2345235](https://github.com/continue-revolution/sd-webui-animatediff/assets/77060047/4372609f-6ad1-48c7-a49d-4da19894fb48)

**Comment by zappityzap** [comment]
Hmm, I'm not sure then. It's trying to import a python module which comes with Automatic1111 that should be there, but isn't. The file for the network module should be `stable-diffusion-webui/extensions-builtin/Lora/network.py`

Some of those extensions could be causing problems, or your Automatic1111 install could be damaged. I'd try updating Automatic1111 first, then try to reproduce with a simple prompt: no lora, no extensions besides AnimateDiff. Might even try disabling the other extensions.

**Comment by jlpintado** [comment]
I have previously tried deactivating all the extensions and it didn't work, I don't like the idea of practically reinstalling automaticc 1111, but apparently there is no other way, when I have the time I will do it and comment if it solves the problem.

**Comment by continue-revolution** [comment]
I have no idea why you cannot import lora. You may want to check if stable-diffusion-webui/extensions-builtin/Lora/network.py exists in your system.

**Comment by jlpintado** [comment]
If the network.py file exists
![q2412r](https://github.com/continue-revolution/sd-webui-animatediff/assets/77060047/d40a9460-031e-4a05-9d5d-4065967acb17)

**Comment by jlpintado** [comment]
The problem has been solved, but I had to reinstall everything automatic 11111.
Once automatic was installed, I installed xformer, then the animediff extension and then control net, and now it works.
I never knew what caused the original problem, but now it works.
Thank you all very much for your advice and guidance.
![tw43234253](https://github.com/continue-revolution/sd-webui-animatediff/assets/77060047/352a2cae-e0f9-464d-bb98-e69303f58b91)
